test/unit/spec/appWidgets/input/textareaInputSpec.js

```diff
@@ -29,12 +29,12 @@ define(['common/runtime', 'widgets/appWidgets2/input/textareaInput', 'testUtil']
         };
     }
 
-    function startWidgetAndSetTextarea(widget, container, inputText) {
-        widget.start({ node: container }).then(() => {
-            const inputElem = container.querySelector('textarea');
-            inputElem.value = inputText;
-            inputElem.dispatchEvent(new Event('change'));
-        });
+    async function startWidgetAndSetTextarea(widget, container, inputText) {
```

Comment:
Gave this module the same treatment as `textInputSpec`. It neede some work anyway, and wasn't much effort to convert the `done` idiom -> `async/await`